spec/controllers/admin/users_controller_spec.rb

```diff
@@ -146,5 +146,27 @@ module Pageflow
         expect(user.reload.account).to eq(other_account)
       end
     end
+
+    describe '#delete_me' do
+      it 'allows to destroy the user when authorize_user_deletion true' do
```

Comment:
I'd rephrase that to "by default". That way we do not leak to much implementation detail into spec description. The actual default setup might change in the future.